numerics/polynomial_evaluators.hpp

```diff
@@ -27,8 +27,11 @@ struct EstrinEvaluator {
       degree,
       internal_polynomial_evaluators::EstrinEvaluator>::Coefficients;
 
+  template<bool fma>
```

Comment:
I would put the `fma` bit in the main template parameter part of `EstrinEvaluator` (same for Horner).  No point in multiplying the levels of templating, it's not like the syntax is beautiful.

Reply:
The purpose is to hide this implementation detail from the signature of the evaluator class, which is part of the API.

Reply:
Ah good point.  That would be ugly.